website/src/content/docs/docs/howtos/Generate client libraries/06types.mdx

```diff
@@ -703,14 +703,6 @@ public final class Animal implements JsonSerializable<Animal> {
 <ClientTabs>
 <ClientTabItem lang="typespec" >
 
-```typespec
-model Animal {
-  name: string;
-  kind: string;
-  ...Record<string | int32>;
-}
-```
-
```

Comment:
this is to demonstrate there are two ways to write additional properties with union types. do we need to delete it?